modules/plugin/coverage-multidim/coverage-api/pom.xml

```diff
@@ -121,7 +121,7 @@
       <dependency>
           <groupId>com.github.spotbugs</groupId>
           <artifactId>spotbugs-annotations</artifactId>
-          <version>RELEASE</version>
+          <version>4.0.1</version>
```

Comment:
We should really manage this in root dependency management section rather than here.

Reply:
I thought RELEASE was removed from Maven 3.   I can revert it or keep it there its up to you.
https://cwiki.apache.org/confluence/display/MAVEN/Maven+3.x+Compatibility+Notes#Maven3.xCompatibilityNotes-PluginMetaversionResolution. Once again my IDE pointed out to me.   If it works do we want the project to possibly use a different version if build later in time?  Maybe for this one we do. Again let me know if you like me to revert this or not.

Reply:
I am happy to explicitly depend on 4.0.1, it is just we should put that in a dependency management section in the root pom.xml

Reply:
I added a dependency management for version 4.0.2 in the root pom.xml.  I also updated the version of the spotbugs-maven-plugin to match the spotbugs-annotations' version.  I build with the profile -Pspotbugs enabled and it completed with the new plugin version.